cmd/ipfswatch/main.go

```diff
@@ -99,7 +100,7 @@ func run(ipfsPath, watchPath string) error {
 	}
 
 	interrupts := make(chan os.Signal)
-	signal.Notify(interrupts, os.Interrupt, os.Kill)
+	signal.Notify(interrupts, os.Interrupt, syscall.SIGTERM)
```

Comment:
the error message is: `os.Kill cannot be trapped (did you mean syscall.SIGTERM?) (SA1016)` where SA1016 -> Trapping a signal that cannot be trapped